contrib/completion/zsh/_docker

```diff
@@ -1955,6 +1955,7 @@ __docker_service_subcommand() {
         "($help)--health-retries=[Consecutive failures needed to report unhealthy]:retries:(1 2 3 4 5)"
         "($help)--health-timeout=[Maximum time to allow one check to run]:time: "
         "($help)--hostname=[Service container hostname]:hostname: " \
+        "($help)--isolation=[Container isolation mode]:isolation:(default process hyperv)" \
```

Comment:
Should this be the same description as the actual flag? ("Isolation mode for the service") ?

Reply:
Or "Service container isolation mode"